environments/dev-env/README.md

```diff
@@ -0,0 +1,395 @@
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
```

Comment:
As a machine owner, I am not eager to run a script without know what it will do.  I would be more willing if it asked for confirmation of each change before it makes it, explaining what it was going to do (e.g., something like displaying the following commands (which I see in the script) and asking for approval:
```sh
        brew update
        curl -O https://dl.google.com/go/go1.19.2.darwin-amd64.pkg
        tar -xvf go1.19.2.darwin-amd64.pkg
        sudo mv go  /usr/bin/
        sudo echo 'export GOROOT=/usr/bin/go' >> ~/.bashrc 
        sudo echo 'export PATH=$PATH:$GOROOT/bin' >> ~/.bashrc 
```